content/security/encryption-hashing.md

```diff
@@ -59,20 +59,21 @@ Once the installation is complete, you can use the `hash` function, as follows:
 ```typescript
 import * as bcrypt from 'bcrypt';
 
-const saltOrRounds = 10;
+const saltRounds = 10;
+const salt = await bcrypt.genSalt(saltRounds);
 const password = 'random_password';
-const hash = await bcrypt.hash(password, saltOrRounds);
```

Comment:
Bcrypt's `hash` takes in a salt or a number of rounds for the second argument. If a number is posted, bcrypt will generate a salt for the hash for you. The original code was fine as is